src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc_NavigationBar.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess
+{
+    internal partial class Editor_InProc
+    {
+        public string GetSelectedNavBarItem(int comboBoxIndex)
+            =>  ExecuteOnActiveView(v => GetNavigationBarComboBoxes(v)[comboBoxIndex].SelectedItem?.ToString());
+
+        public string[] GetNavBarItems(int comboBoxIndex)
+            => ExecuteOnActiveView(v =>
+                GetNavigationBarComboBoxes(v)[comboBoxIndex]
+                .Items
+                .OfType<object>()
+                .Select(i => i?.ToString() ?? "")
+                .ToArray());
+
+        public int GetNavbarItemIndex(int index, string itemText)
+        {
+            int FindItem(ComboBox comboBox)
+            {
+                for (int i = 0; i < comboBox.Items.Count; i++)
+                {
+                    if (comboBox.Items[i].ToString() == itemText)
+                    {
+                        return i;
+                    }
+                }
+
+                return -1;
+            }
+
+            return ExecuteOnActiveView(v => FindItem(GetNavigationBarComboBoxes(v)[index]));
+        }
+
+        public void ExpandNavigationBar(int index)
+        {
+            ExecuteOnActiveView(v =>
+            {
+                var combobox = GetNavigationBarComboBoxes(v)[index];
+                combobox.Focus();
+                combobox.IsDropDownOpen = true;
+            });
+        }
+
+        public void SelectNavBarItem(int comboboxIndex, string selection)
+        {
+            var itemIndex = GetNavbarItemIndex(comboboxIndex, selection);
+            if (itemIndex < 0)
+            {
+                throw new ArgumentException($"Could not find {selection} in combobox");
+            }
+
+            ExpandNavigationBar(comboboxIndex);
+            System.Windows.Forms.SendKeys.SendWait("{HOME}");
```

Comment:
@dpoeschl No, we should be using our own helper here. Why the use of this here @rchande?